maestro/trainer/models/paligemma_2/core.py

```diff
@@ -201,8 +201,11 @@ def train(config: PaliGemma2Configuration | dict) -> None:
         optimization_strategy=OptimizationStrategy(config.optimization_strategy),
         cache_dir=config.cache_dir,
     )
+    dataset_location = resolve_dataset_path(config.dataset)
+    if dataset_location is None:
```

Comment:
`if dataset_location is not None` and indent code below and return after if-block.